experimental/traffic-portal/server.config.ts

```diff
@@ -202,6 +204,12 @@ function isConfig(c: unknown): c is ServerConfig {
 	if (typeof((c as {trafficOps: unknown}).trafficOps) !== "string") {
 		throw new Error("'trafficOps' must be a string");
 	}
+	if (!Object.prototype.hasOwnProperty.call(c, "tpv1Url")) {
+		throw new Error("'tpv1Url' is required");
+	}
+	if (typeof((c as {tpv1Url: unknown}).tpv1Url) !== "string") {
+		throw new Error("'tpv1Url' must be a string");
+	}
```

Comment:
These blocks can be combined into a single check that doesn't need `as` if you import `hasProperty` from our utils package and use that. Probably also occurs elsewhere in the PR, and definitely in the code at large.